test/reference_tests/mermaid_legolas.md

```diff
@@ -19,9 +19,9 @@ subgraph STEP_B[Step b]
   direction TB
   subgraph STEP_B_InputSchema[Input: SchemaRadV1]
     direction RL
-    STEP_B_InputSchemalist{{"list::Array"}}
+    STEP_B_InputSchemalist{{"list::Vector{Int64}"}}
     class STEP_B_InputSchemalist classSpecField
-    STEP_B_InputSchemafoo{{"foo::String"}}
+    STEP_B_InputSchemafoo{{"foo::Union{Missing, String}"}}
```

Comment:
nb: i'm surprised that there's a space here, and that it isn't `Union{Missing,String}`

Reply:
yeah, and that feels like something Julia may decide to break in the future :/

Reply:
womp womp